tutorials/Tutorial 5 - Plotting.ipynb

```diff
@@ -108,13 +108,13 @@
     "cmap = plt.cm.Paired\n",
```

Comment:
Okay, I looked into this and fixed it. The reason why it still runs (the incorrect args simply do nothing) is because we have **kwargs as an argument to update the default drawing settings (it was too wordy to write each setting out) and so it can accept any kwarg. If it doesn't mean anything, however, nothing will happen.